casper/src/main/resources/MultiSigRevVault.rho

```diff
@@ -199,7 +199,7 @@ in {
                                     @(*multiSig, *nonceMapStore)!(nonceMap) |
                                     ret!((false, "given data does not match stored data"))
                                   } else {
-                                    if (((not unsealed) and nonceConfirmers.contains(auth)) or
```

Comment:
Just curious. What is the side effect of using `auth`? 
Is it because one person could create several different auth token and make the proposal pass?

Reply:
Auth key is created with the secret which is related to specific public key. So if the same deployer creates multiple auth keys they are all valid to create/confirm transfer, although they will be different unforgeable names.